src/components/Spawn/editHostModal/FieldTemplates/ExpirationRow.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+import { ObjectFieldTemplateProps } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { form } from "pages/projectSettings/tabs/utils";
+
+const { getFields } = form;
```

Comment:
So sorry I didn't know this in past code reviews. This should be moved to a more global directory. Possibly In a `spruceform/utils` directory or something along those lines.